packages/components/app/styles/components/tag.scss

```diff
@@ -0,0 +1,101 @@
+//
+// TAG COMPONENT
+//
+// properties within each class are sorted alphabetically
+//
+//
+
+@use "../mixins/focus-ring" as *;
+
+$hds-tag-border-radius: 13px;
```

Comment:
@alex-ju When the content wraps, 13px doesn't keep the pill shape...thinking if you bump it up to something ridiculous like 50px that should cover _most_ of the edge cases.